modules/cert.cpp

```diff
@@ -46,7 +46,7 @@ class CCertMod : public CModule {
 		AddHelpCommand();
 		AddCommand("delete", static_cast<CModCommand::ModCmdFunc>(&CCertMod::Delete),
 			"", "Delete the current certificate");
-		AddCommand("info", static_cast<CModCommand::ModCmdFunc>(&CCertMod::Info));
+		AddCommand("info", static_cast<CModCommand::ModCmdFunc>(&CCertMod::Info)),"", "Shows whether or not you have a certificate";
```

Comment:
AddCommand("info", static_cast<CModCommand::ModCmdFunc>(&CCertMod::Info), "", "Shows whether or not you have a certificate");